### PR TITLE
remove positive state enforcement

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**
Hi,
Following https://github.com/SeisSol/SeisSol/discussions/746 (while the discussion did not really come to a conclusion)
I propose removing the max on the state variable and documenting this change.
Note that sem2dpack does not enforce positivity of the state variable either:
https://github.com/jpampuero/sem2dpack/blob/master/SRC/bc_dynflt_rsf.f90
I would create v1.1.0 SeisSol afterward (tagging the version).

It should not require recomputing a solution of tpv103, as the state never gets negative in this benchmark.

### CODE REVIEW

**Comment by sebwolf-de** [comment]
How does this influence #668?

**Comment by Thomas-Ulrich** [comment]
> How does this influence #668?

this does not fix it.

**Comment by Thomas-Ulrich** [comment]
But I think it then makes sense to allow negative initial states (But then flood the user with warnings :-))

**Comment by sebwolf-de** [comment]
Okay, for #668 we also came to the conclusion that a zero traction is unphyiscal in the R&S framework, that's why we needed the omega > 0...

File: Documentation/dynamic-rupture.rst [hunk 1]
```diff
@@ -282,7 +282,9 @@ with
   \end{aligned}.
 
 
-
+Note that between commits b014d1180d53a5231e92bb5062a45711fea9aade (merging #306) of March 17th, 2021, and v1.1.0, the state variable was enforced positive in this friction law.
```
Comment (sebwolf-de) [inline Documentation/dynamic-rupture.rst:285]: I wouldn't hard code the versions here, just mention the PRs (#306 and #752).

**Review by sebwolf-de — COMMENTED** [review]
LGTM, but I wouldn't add a v1.1.0 yet. After all we haven't really released v1.0.0 yet, so this would be v1.0.0-rc2 probably.

**Comment by Thomas-Ulrich** [comment]
> v1.0.0-rc2

So what feature is needed to release v1.0.0?

**Review by sebwolf-de — APPROVED** [review]

**Comment by sebwolf-de** [comment]
Push new references to the `precomputed-seissol` repository.
* Checkout https://github.com/7bits-register/precomputed-seissol
* Run TPV105 again
* Check if output is reasonable.
* Update https://github.com/7bits-register/precomputed-seissol/tree/master/tpv105/precomputed/double (+ single directory)
* Push
* Add commit id https://github.com/SeisSol/SeisSol/blob/master/gitlab-cpu-pipeline.yml#L113 and https://github.com/SeisSol/SeisSol/blob/master/gitlab-gpu-pipeline.yml#L149

**Comment by Thomas-Ulrich** [comment]
I just check tpv105-3D, and the state indeed gets negative but on a very narrow zone at shallow depth (e.g. state variable, saturated colormap, t=9s)
image

**Comment by sebwolf-de** [comment]
Huh, that's interesting. But the CI only runs for 2s, so this should not be the problem..

**Comment by sebwolf-de** [comment]
I compared the CI output:
With positive state variable enforced:
Screenshot from 2022-12-15 10-48-09
With negative state variable allowed:
Screenshot from 2022-12-15 10-48-05
Note that both images share the same colorbar, in the upper picture, the blue circle is exactly at zero, in the lower picture, the blue circle is slightly below zero.

**Comment by Thomas-Ulrich** [comment]
Yes, I ve compared the folder, and I only found this problem:
https://github.com/SeisSol/Examples/pull/57
Else the inputs are similar (well SlipRateOutputType=0 in the CI, but I checked that it is not the problem).
So at the end, I think the state is negative because of the coarse resolution.

**Pull request merged**